styles/OurClients.module.css

```diff
@@ -0,0 +1,30 @@
+.our_clients {
+  margin-top: 64px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 0px;
+  border-bottom: 1px solid black;
+  padding-bottom: 64px;
+}
+
+.our_clients__text {
+  font-family: "Roboto Mono";
+  font-style: normal;
+  font-weight: 400;
+  width: 105px;
+  line-height: 100%;
+}
+.our_clients__icons {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0px;
+  gap: 94px;
+  padding-left: 64px;
```

Comment:
Same here. Writing padding in 1 line is better than overriding it, especially if you override it after other properties. It might lead to a hard-to-find property situation.